### ISSUE TITLE
how to use multi gpu with one single machine on docker?

### ISSUE THREAD

**Issue by beyondboy** [facebookresearch/wav2letter#138]
I have 8 gpu kard in my single machin, for some reason i only can use it  with docker, and i use mpirun as follows:
### mpirun -n 8  ./Train train -enable_distributed true --flagsfile train.cfg

but i got errs as follows:
```
The value of the MCA parameter "plm_rsh_agent" was set to a path
that could not be found:

  plm_rsh_agent: ssh : rsh

Please either unset the parameter, or check that the path is correct
[4665-asr-w2l-1547109629:219410] [[INVALID],INVALID] FORCE-TERMINATE AT Not found:-13 - error plm_rsh_component.c(327)
[4665-asr-w2l-1547109629:219410] *** Process received signal ***
[4665-asr-w2l-1547109629:219410] Signal: Segmentation fault (11)
[4665-asr-w2l-1547109629:219410] Signal code: Address not mapped (1)
[4665-asr-w2l-1547109629:219410] Failing at address: (nil)
[4665-asr-w2l-1547109629:219410] [ 0] /lib/x86_64-linux-gnu/libc.so.6(+0x3ef20)[0x7f3216853f20]
[4665-asr-w2l-1547109629:219410] *** End of error message ***
```
can you give me some advices about how to fix this?

**Comment by jacobkahn** [comment]
@beyondboy — a quick Google search points to this being an issue with `ssh` not being installed on your system ([take a look here](https://github.com/open-mpi/ompi/issues/3625)).

If installing `ssh` doesn't work, can you tell me what do you see when you run [`echo $NVIDIA_VISIBLE_DEVICES`](https://github.com/NVIDIA/nvidia-docker/wiki/Frequently-Asked-Questions#i-have-multiple-gpu-devices-how-can-i-isolate-them-between-my-containers)?

**Comment by beyondboy** [comment]
after installing ssh, it works!  thanks!

btw: i run the librispeech tutorial , if using 8 gpu and set batch_size=64, how should set the learning rate?(original batch_size=4 lr=0.1) increasing lr linearly with ratio to batch_size and gpu num?
